Microsoft.Identity.Web/ClaimsPrincipalExtensions.cs

```diff
@@ -63,6 +82,32 @@ public static string GetTenantId(this ClaimsPrincipal claimsPrincipal)
             return tenantId;
         }
 
+        /// <summary>
+        /// Gets the Policy Id associated with the <see cref="ClaimsPrincipal"/>
+        /// </summary>
+        /// <param name="claimsPrincipal">the <see cref="ClaimsPrincipal"/> from which to retrieve the policy id</param>
+        /// <returns>Policy Id of the identity, or <c>null</c> if it cannot be found</returns>
+        public static string GetPolicyId(this ClaimsPrincipal claimsPrincipal)
+        {
+            string policyId = claimsPrincipal.FindFirstValue(ClaimConstants.Tfp);
```

Comment:
>Tfp) [](start = 76, length = 4)

? -> why are we relying on tfp here? it's not required.

Reply:
@jennyf19 MSAL uses `tfp` on B2C `IAccount`, so we need it to use methods like `GetAccountAsync` where it receives the accountIdentifier as param.

Reply:
@TiagoBrenck let's discuss this too...i think we can make this better on the MSAL side.